### ISSUE TITLE
Organization name in navigation bar has insufficient color contrast

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4083]
## Area or Component
Navigation bar - main application

## Quick Summary
The organization name in the navigation bar (under "SimpleReport") doesn't have enough contrast with the background (it's light grey on white.)

## Accessibility Violation
Elements must have sufficient color contrast: https://dequeuniversity.com/rules/axe/4.4/color-contrast?application=AxeChrome

## Action Requested
Change the color of the organization name to be a darker color and have improved contrast.

## Screenshots (optional)
"Dev Camp" in this screenshot:
![Uploading Screen Shot 2022-07-26 at 4.10.13 PM.png…]()

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=1978534823&range=12:12

**Comment by emmastephenson** [comment]
Just kidding - this was a false positive from the report
